rspace/src/test/scala/coop/rchain/rspace/test/InMemoryStore.scala

```diff
@@ -183,11 +186,8 @@ object InMemoryStore {
       case Right(value) => value
     }
 
-  def create[C, P, A, K <: Serializable](implicit sc: Serialize[C],
-                                         sk: Serialize[K]): InMemoryStore[C, P, A, K] =
+  def apply[C, P, A, K <: Serializable]()(implicit sc: Serialize[C],
```

Comment:
For now, let's stick with `create` for parity's sake.

I have mixed feelings about instantiating stateful objects with `apply`...